addons/sys_list/fnc_displayHint.sqf

```diff
@@ -19,10 +23,26 @@
 GVAR(hintTitle) = _this select 0;
 GVAR(hintLine1) = _this select 1;
 GVAR(hintLine2) = _this select 2;
-GVAR(hintDuration) = 1;
-if ((count _this) == 4) then {
-    GVAR(hintDuration) = _this select 3;
+private _hintDuration = -1;
```

Comment:
Should we change these checks to params and default values
`params ["_hintTitle", "_hintLine1", "_hintLine2", ["_hintDuration", -1], ["_hintColor",[1, 0.8, 0, 1]]];`
?

Reply:
Is it more efficient to to params first and assign it to GVARs later on than to assign it to GVARs directly?
```
params ["_hintTitle", "_hintLine1", "_hintLine2", ["_hintDuration", -1], ["_hintColor",[1, 0.8, 0, 1]]];

GVAR(hintTitle) = _hintTitle;
GVAR(hintLine1) = _hintLine1;
GVAR(hintLine2) = _hintLine2;
GVAR(hintColor) = _hintColor;
```

Reply:
From slack discussion, we use params.